package.json

```diff
@@ -196,6 +196,25 @@
                 "title": "Makefile: Reset the Makefile Tools Extension workspace state (For troubleshooting)"
             }
         ],
+        "problemMatchers": [
+            {
+              "name": "gcc",
```

Comment:
Will this clash with the C++ extension's problem matcher?  I thought we would just insert the correct regex before starting the task if someone used `$gcc` as their problem matcher. Otherwise, I would worry that vscode doesn't know what to do with two `$gcc` entries.

Reply:
Oh, I forgot to reply to this one. According to what I read in VSCode problem matcher documentations (and also in issue comment https://github.com/microsoft/vscode/issues/61497#issuecomment-456320050), the last one that is parsed wins.
On one side, who uses Makefile Tools without CppTools? So maybe we should remove gcc from package.json.
But if that happens, having one gcc here as a backup doesn't hurt. Let's discuss in the morning meeting.

Reply:
If last one wins and VS Code doesn't log errors/warnings, then we can keep this one.  There are actually quite a few Makefile Tools users who do not use cpptools (14%)